modules/evm-bridge/src/lib.rs

```diff
@@ -72,6 +74,89 @@ pub mod module {
 }
 
 impl<T: Config> EVMBridgeTrait<AccountIdOf<T>, BalanceOf<T>> for Pallet<T> {
+	// Calls the name method on an ERC20 contract using the given context
+	// and returns the token name.
+	fn name(context: InvokeContext) -> Result<Vec<u8>, DispatchError> {
+		// ERC20.name method hash
+		let input = hex!("06fdde03").to_vec();
```

Comment:
The magic hashes should be defined as const, and unit tests are needed to explicitly reproduce the hash.